docs/src/main/asciidoc/pubsub.adoc

```diff
@@ -325,6 +347,13 @@ After this amount of time has elapsed (counting from the first element added), t
 Enables batching.|No | false
 |===
 
-=== Sample
+=== Details
```

Comment:
I think this Details section might not be necessary because they are very abstracted from the user; someone wanting to truly understand would look at the sources instead. For me, the details about the factories went over my head - I think newer users of this library might be in a similar boat.